### PR TITLE
(optional) app.pid file to override default Enforce process check

### PR DESCRIPTION
**Pull request by timcerexhe**

### CODE REVIEW

File: testapp/start-haproxy.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+cd `dirname $0`
+
+FRONT="$1"
+APP="$2"
+
+echo "WOO!"
+pwd
```
Comment (latos) [inline testapp/start-haproxy.sh]: cleanup

File: testapp/start-haproxy.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```
Comment (latos) [inline testapp/start-haproxy.sh]: document what the script is for? i imagine this is an unrelated yakshave...

File: integration_test.go [hunk 3]
```diff
@@ -183,6 +188,49 @@ func TestRun(t *testing.T) {
...
+		fmt.Printf(" - %s on %d @ %d - %s\n", d.Id, d.Port, d.Pid, d)
+		//deploy commit 
+		if regexp.MustCompile("\\d\\d\\d\\d-\\d\\d-\\d\\d-\\d\\d-\\d\\d-\\d\\d").MatchString(d.Id) {
+			if node != nil {
+				t.Fatalf("multiple node deploys running %s and %s", d.Id, node.Id)
+			}
+			node = d
+		} else if regexp.MustCompile("haproxy-\\d+").MatchString(d.Id) {
+			if haproxy != nil {
+				t.Fatalf("multiple haproxy instances running %s and %s", d.Id, haproxy.Id)
```
Comment (latos) [inline integration_test.go]: can you reword "haproxy" to "testapp haproxy" because otherwise confusion with camus' haproxy instance

**Comment by timcerexhe** [comment]
ping

now with checking in list()

**Comment by latos** [comment]
LGTM
